videoframes.py

```diff
@@ -206,6 +221,22 @@ def change_stream_quality(self, quality):
         self.stream.change_stream_quality(quality)
         self.player.play()
 
+    def open_stream_in_browser(self, event):
+        url = self.stream.url.lower()
+        url = ("http://www." if "www" not in url else "http://") + url
+        if "twitch" in url:
+            url = url + "/chat"
+        os = platform.system().lower()
```

Comment:
There's no need to cast to lower() anymore, I've changed the code in the enum class :)